deploy/gcp/README.md

```diff
@@ -0,0 +1,99 @@
+# Deploy TiDB Operator and TiDB cluster on GCP GKE
+
+## Requirements:
+* [gcloud](https://cloud.google.com/sdk/install)
+* [terraform](https://www.terraform.io/downloads.html)
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl) >= 1.11
+* [helm](https://github.com/helm/helm/blob/master/docs/install.md#installing-the-helm-client) >= 2.9.0
+* [jq](https://stedolan.github.io/jq/download/)
+
+## Configure gcloud
+
+https://cloud.google.com/sdk/docs/initializing
+
+## Setup
+
+The default setup will create a new VPC, two subnetworks, and an f1-micro instance as a bastion machine. The GKE cluster is created with the following instance types as worker nodes:
+
+* 3 n1-standard-4 instances for PD
+* 3 n1-highmem-8 instances for TiKV
+* 3 n1-standard-16 instances for TiDB
+* 3 n1-standard-2 instances for monitor
```

Comment:
This is because GKE creates 3 nodes even when you only want one? I think we should add the gcloud api requests needed to delete the unused nodes (the scaler does this automatically using the go library). It looks like there is an unused node for TiDB also. I think TiDB and monitor should get re-scheduled if their node is removed.